Documentation/FrameworkOverview.md

```diff
@@ -20,21 +20,21 @@ of a long-running operation. In any case, something generates the events and sen
 `Event` is an enumerated type representing either a value or one of three
 terminal events:
 
- * The `Next` event provides a new value from the source.
- * The `Failed` event indicates that an error occurred before the signal could
+ * The `value` event provides a new value from the source.
+ * The `failed` event indicates that an error occurred before the signal could
    finish. Events are parameterized by an `ErrorType`, which determines the kind
    of failure that’s permitted to appear in the event. If a failure is not
    permitted, the event can use type `NoError` to prevent any from being
    provided.
- * The `Completed` event indicates that the signal finished successfully, and
+ * The `completed` event indicates that the signal finished successfully, and
    that no more values will be sent by the source.
- * The `Interrupted` event indicates that the signal has terminated due to
+ * The `interrupted` event indicates that the signal has terminated due to
    cancellation, meaning that the operation was neither successful nor
    unsuccessful.
 
 ## Signals
 
-A **signal**, represented by the [`Signal`][Signal] type, is any series of [events](#events)
+A **signal**, represented by the [`SignalProtocol`][Signal] protocol, is any series of [events](#events)
```

Comment:
This should not be changed. We use `SignalProtocol` only to constraint the extensions. A signal is always the concrete type `Signal`.